vagrant/celery.service

```diff
@@ -1,13 +1,14 @@
 [Unit]
 Description=Celery Service
-After=network.target
+After=network.target redis.service
 
 [Service]
 Type=forking
 User=vagrant
 Group=vagrant
 EnvironmentFile=-/etc/celery.conf
 WorkingDirectory=/var/lib/celery
+ExecStartPre=/bin/sleep 30
```

Comment:
This seem to be a bit fragile. Is this to wait for Redis to start up?

Reply:
Yes, I am looking for a nice way. It was the result of my Dell E4310.
Do you have any good idea?

Reply:
Sorry for the delay, but I think we can drop the sleep here seeing we adding the wait for redis.service. WDYT?

Reply:
I leave this to you.
There is no problem if you can successfully start the timesketch.